src/lightning_app/utilities/packaging/cloud_compute.py

```diff
@@ -80,6 +80,12 @@ def __post_init__(self) -> None:
 
         self.name = self.name.lower()
 
+        if self.shm_size is None:
+            if "gpu" in self.name:
+                self.shm_size = 1024
+            else:
+                self.shm_size = 0
```

Comment:
We had 64 on standard works earlier, does 0 gets us 64 by default?

Reply:
Yes. I didn't change the behaviour.